stubs/redis/redis/ocsp.pyi

```diff
@@ -1,13 +1,21 @@
-from typing import Any
+from _typeshed import Incomplete
+from ssl import SSLObject, SSLSocket
+from typing_extensions import Literal
+
+from cryptography.x509.base import Certificate
+from OpenSSL.SSL import Connection
+
+def ocsp_staple_verifier(con: Connection, ocsp_bytes: bytes, expected: bytes | None = ...) -> Literal[True]: ...
 
 class OCSPVerifier:
-    SOCK: Any
-    HOST: Any
-    PORT: Any
-    CA_CERTS: Any
-    def __init__(self, sock, host, port, ca_certs: Any | None = ...) -> None: ...
-    def components_from_socket(self): ...
-    def components_from_direct_connection(self): ...
-    def build_certificate_url(self, server, cert, issuer_cert): ...
-    def check_certificate(self, server, cert, issuer_url): ...
-    def is_valid(self): ...
+    SOCK: SSLObject
+    HOST: str
+    PORT: int
+    CA_CERTS: str | None
+    def __init__(self, sock: SSLObject | SSLSocket, host: str, port: int, ca_certs: str | None = ...) -> None: ...
```

Comment:
Why is the type for `sock` in the `__init__` method different to the attribute type?

Reply:
I updated one but forgot the other.